src/Dev/Tasks/MigrateFileTask.php

```diff
@@ -52,6 +55,12 @@ class MigrateFileTask extends BuildTask
     /** @var Logger */
     private $logger;
 
+    public function __construct()
+    {
+        Deprecation::notice('4.12.0', 'Will be removed without equivalent functionality to replace it', Deprecation::SCOPE_CLASS);
```

Comment:
Why we don't use function wrapper `Deprecation::withNoReplacement` in this case?

Reply:
This won't get automatically called so there no need to.  Because it's one-off task designed for CMS 3 file migrations, it's extremely unlikely that this would get run at the same time that Deprecation notices have been enabled